packages/storybook/stories/va-back-to-top.stories.jsx

```diff
@@ -43,7 +43,7 @@ const Template = () => {
           height: '500px',
           display: 'flex',
           justifyContent: 'center',
-          background: '#112E51',
+          background: '#162e51',
```

Comment:
If this color is available in the CSS Library, we should probably change this to the corresponding token instead of hard coded hex value: `var(--color-primary-darker)`

Reply:
Yeah good idea! Updated: 

![image](https://github.com/department-of-veterans-affairs/component-library/assets/15097156/d1d5a562-fe4e-48d7-8cf7-002f3683f1eb)